src/components/MediaPlayer/VideoJS/components/js/VideoJSTrackScrubber.js

```diff
@@ -201,19 +208,25 @@ function TrackScrubberButton({ player, trackScrubberRef, timeToolRef }) {
 	 */
 	const timeUpdateHandler = () => {
 		if (player.isDisposed()) return;
```

Comment:
I did not think to test the track scrubber when I was looking at the issue with the flashing struct nav. Just to be safe that we don't accidentally reintroduce that behavior, we should probably have this time update return on ended as well.
```suggestion
		if (player.isDisposed() || player.ended()) return;
```